docs/package.json

```diff
@@ -28,6 +30,7 @@
         "eleventy-plugin-reading-time": "^0.0.1",
         "imagemin": "^8.0.1",
         "imagemin-cli": "^7.0.0",
+        "luxon": "^2.4.0",
```

Comment:
what is this used for?

Reply:
This is a library we use to format dates and times.

https://github.com/eslint/eslint/blob/6210ff7debf11c96441185aeef786ebf84a78ee5/docs/.eleventy.js#L12-L14

https://github.com/eslint/eslint/blob/6210ff7debf11c96441185aeef786ebf84a78ee5/docs/.eleventy.js#L57-L64

We're using it directly, but it wasn't declared as a dependency (the package was installed because it's a dependency of another dependency we're using), so [node/no-extraneous-require](https://github.com/mysticatea/eslint-plugin-node/blob/HEAD/docs/rules/no-extraneous-require.md) was reporting error on `require("luxon")`.